addons/mail/static/src/models/message_seen_indicator/message_seen_indicator.js

```diff
@@ -239,11 +223,46 @@ function factory(dependencies) {
             }
             return [['replace', otherPartnersThatHaveSeen]];
         }
+
+        /**
+         * @private
+         * @returns {mail.message|undefined}
```

Comment:
undefined should not be allowed (and it will crash, can't insert/link undefined) (-> messageId should be required field)
no need to change code, just don't add undefined in docstring